script.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable no-use-before-define */
+const saveBtn = document.querySelector('#addBtn');// eslint-disable-line no-unused-vars
+const bookListDiv = document.querySelector('#bookList');
+dataStorage(JSON.parse(localStorage.getItem('books'))); // eslint-disable-line no-undef
+function addBook() {
+  const bookList = JSON.parse(localStorage.getItem('books')) === null ? [] : JSON.parse(localStorage.getItem('books'));
+  const book = {
+    title: document.querySelector('#inputTitle').value,
+    author: document.querySelector('#inputAuthor').value,
+  };
+
+  bookList.push(book);
+  localStorage.setItem('books', JSON.stringify(bookList));
+  dataStorage(bookList); // eslint-disable-line no-undef
+}
+
+addBtn.addEventListener('click', addBook); // eslint-disable-line no-undef
+
+function removeBook(index) {
+  const bookList = JSON.parse(localStorage.getItem('books')) === null ? [] : JSON.parse(localStorage.getItem('books'));
+  bookList.splice(index, 1);
+
+  if (localStorage.getItem('books') === null) {
+    localStorage.setItem('books', JSON.stringify(bookList));
+  } else {
+    localStorage.setItem('books', JSON.stringify(bookList));
+  }
```

Comment:
- [ ] I can't able to add and remove a book. I'm getting this error in the console, Please fix this.

![Screenshot from 2021-10-19 01-22-12](https://user-images.githubusercontent.com/22434660/137797676-9decd703-a081-48bb-9b45-88d3edf0cd2c.png)